### DISCUSSION TITLE
color result of the taken pics

### DISCUSSION THREAD

**Discussion by mathiasonia** [photobooth-app/photobooth-app#243] [General]
I want to be sure,I take some pics to see how the photobooth works and I don't know if t was the camera module V3 or another issue but the color of the pics are not like the natural one 

I don't know how I can make one to show, if you have a idea

**Comment by mgineer85** [comment]
You can use the libcamera apps to test. I think the camera could be faulty or the cabling.
Except from filters there is no change in colors. Maybe you enabled the pilgram filter?

**Reply by mathiasonia** [reply]
I don't use any filter to see how the app works

**Comment by mathiasonia** [comment]
Hi
I try what you say about the wiring of the camera and I make some test after maybe with an other ambient light but at my lunch pause I try to make some picture and I found like the links after ,the black are little purple or pink

and finally it can be my NoIR camera the problem because the sensitivity was not the same as the links explain

https://forums.raspberrypi.com/viewtopic.php?t=285519

&

https://forums.raspberrypi.com/viewtopic.php?t=245994

I will buy a normal camera v3 with IR filter to test

**Reply by mgineer85** [reply]
Sorry, I never used a NoIR camera and don't think its a good idea for a photobooth actually... https://github.com/waveform80/picamera/issues/137
